.circleci/src/jobs/alert-slack-push-success.yml

```diff
@@ -109,14 +109,20 @@ steps:
           fi
 
 
-          # Generate git diff for audius-docker-compose commits from last "release-vX.Y.Z auto-deploy" commit to HEAD
+          # Update tags in audius-docker-compose stage branch
           cd ..
           git clone --branch stage https://github.com/AudiusProject/audius-docker-compose.git audius-docker-compose
           cd audius-docker-compose
           last_release_commit=$(git log --format=format:%H --grep='^release-v.*auto-deploy$' -1)
-          GIT_DIFF=$(git log --pretty=format:'%an - %s [<https://github.com/AudiusProject/audius-docker-compose/commit/%H|%h>]' --abbrev-commit $last_release_commit..HEAD | grep -v '^audius-infra - Update tag to' | sed 's/"/\\"/g' | tr -d '\r' | sed ':a;N;$!ba;s/\n/=DELIM@/g')
-          git commit --allow-empty -m "$CIRCLE_BRANCH auto-deploy"
+          sed -i "s/{TAG:-.*}/{TAG:-$CIRCLE_SHA1}/" creator-node/docker-compose*.yml
```

Comment:
this is the same SHA as what the main branch CI sets because the create-release-branch logic pushes the same commit to both main and this new release branch